src/core/zowe/core_for_zowe_sdk/config_file.py

```diff
@@ -118,7 +120,9 @@ def init_from_file(
                 pass
 
         if self.filepath is None or not os.path.isfile(self.filepath):
-            warnings.warn(f"Config file does not exist at {self.filepath}")
+            if not suppress_config_file_warnings:
```

Comment:
Now that we have logging in place, we might be able to remove the `suppress_config_file_warnings` flag and log some of the config file warnings at INFO level instead.

However I think that removing warnings that were already present would be outside the scope of the effort to add a logger, so this could be addressed in a separate issue 🙂

Reply:
I agree with you. It should be addressed in a separate issue.